src/components/NoteDetails/NoteDetailsPage.js

```diff
@@ -12,10 +14,16 @@ const Wrapper = styled.div`
 `;
 
 export const NoteDetailsPage = () => {
+  const [isDialogOpen, setIsDialogOpen] = useState(false);
+
   return (
     <Wrapper>
-      <NoteDetailsTopBar />
+      <NoteDetailsTopBar openDialog={() => setIsDialogOpen(true)} />
```

Comment:
Why don't we just put `DialogDeleteNote` inside the `NoteDetailsTopBar` ?
Why do we have to put it in the `NoteDetailsPage`, and then pass a function as a property to `NoteDetailsTopBar`?
It seems like the code would be simpler if we just had `DialogDeleteNote` rendered inside the `NoteDetailsTopBar`.